config/robots.rb

```diff
@@ -4253,4 +4253,198 @@
       from 'BGE Cap Lib PCR', bed(1)
       to 'BGE Cap Lib PCR XP', bed(9)
     end
+
+    # scRNA pipeline
```

Comment:
```suggestion
    # scRNA Core pipeline
```

Pedantic sorry, but there is another pipeline called scRNA :/